plutus-ir/src/Language/PlutusIR/Compiler.hs

```diff
@@ -1,9 +1,19 @@
+{-# LANGUAGE FlexibleContexts #-}
 module Language.PlutusIR.Compiler (
     compileTerm,
     Compiling,
     CompError (..),
     runCompiling) where
 
+import           Language.PlutusIR
 import           Language.PlutusIR.Compiler.Error
-import           Language.PlutusIR.Compiler.Term
+import qualified Language.PlutusIR.Compiler.Term  as Term
 import           Language.PlutusIR.Compiler.Types
+
+import qualified Language.PlutusCore              as PLC
+
+import           Control.Monad
+
+-- | Compile a 'Term' into a PLC Term. Note: the result *does* have globally unique names.
```

Comment:
The comment is still here.

Reply:
But subtly different ;)

Reply:
Oh, yes :)
Though, I think any thing that runs in `Quote` is assumed to have globally unique names?

Reply:
I think that's a dangerous assumption. I'd prefer to note it when possible... I still wish we had a type-level distinction here.